rundeckapp/grails-spa/packages/ui-trellis/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "A collection of utilities used across various javascript applications and components.",
   "main": "./lib/index.js",
   "scripts": {
-    "test": "jest --no-cache",
+    "test": "npx jest --no-cache",
+    "test:unit": "npx jest",
```

Comment:
since you're running npm install, it should be possible to remove the npx command.

Also, what do you think of adding --silent flag on test:unit?